python/src/aiconfig/editor/client/src/shared/prompt_schemas/OpenAIChatModelParserPromptSchema.ts

```diff
@@ -38,7 +38,7 @@ export const OpenAIChatModelParserPromptSchema: PromptSchema = {
         items: {
           type: "object",
           required: ["name", "parameters"],
-          parameters: {
+          properties: {
```

Comment:
This was using incorrect key, properties is needed:
<img width="669" alt="Screenshot 2024-01-04 at 2 04 50 PM" src="https://github.com/lastmile-ai/aiconfig/assets/5060851/2c85f676-a767-4757-af08-ac80cff9e762">